md/sv2-workshop.md

```diff
@@ -16,38 +16,62 @@ http://75.119.150.111:8888/html/sv2-workshop.html
 ---
 
 ## Prerequisites
-
-### Install Rust:
-```
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-```
-
-### Clone & Build SRI
-
-```
-cd $HOME
-git clone https://github.com/stratum-mining/stratum
-cd stratum
-git checkout workshop
-```
-
----
-
-## Get a release from SV2 Bitcoin Core fork
-
-Install the required `bitcoin-core` fork by building from [Sjors's `sv2-tp-0.1.3` tag](https://github.com/Sjors/bitcoin/tree/sv2-tp-0.1.3):
+The required programs are:
+1. `bitcoin-core` fork with Sv2 support.
+2. `stratum` repo with roles logic.
+3. `cpuminer` to act as a hasher (miner) (Miner Role only).
+
```

Comment:
I think you should put `tmux` also here as a requirement since you're mentioning its usage below